books.js

```diff
@@ -0,0 +1,76 @@
+const bookSection = document.querySelector('.books');
+const bookTitle = document.getElementById('book-title');
+const bookAuthor = document.getElementById('book-author');
+const form = document.getElementById('form');
+
+// Check if books are in LocalStorage
+function getBooks() {
+  let myBooks;
+  if (localStorage.getItem('newBooks') === null) {
+    myBooks = [];
+  } else {
+    myBooks = JSON.parse(localStorage.getItem('newBooks'));
+  }
+  return myBooks;
+}
+
+// Div that contains book HTML
+function showBooks() {
+  bookSection.innerHTML = '';
+  const myBooks = getBooks();
+  for (let i = 0; i < myBooks.length; i += 1) {
+    const book = myBooks[i];
+    const bookList = `
+    <div id="book-info">
+    <p>${book.title}</p>
+    <p>${book.author}</p>
+    <button id=${i} type="submit" class="remove-book" onclick="removeBook(this.id)">Remove</button>
+    <hr> 
+    </div>
+    `;
+    bookSection.innerHTML += bookList;
+  }
+}
+
+// This code clears the input field after books have been submitted
+function clearInput() {
+  bookTitle.value = '';
+  bookAuthor.value = '';
+}
+
+// Add books to local storage
+function setLocalStorage(item) {
+  const books = getBooks();
+  books.push(item);
```

Comment:
- [ ] OPTIONAL
It will be a good practice to favor the use array methods such as `concat` over `push`, that doesn't mutate the array.